AutomatedTesting/Gem/PythonTests/Prefab/tests/prefab_notifications/PrefabNotifications_PropagationNotificationsReceived.py

```diff
@@ -0,0 +1,59 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+propagationBegun = False
+propagationEnded = False
```

Comment:
repeated nit: Can these be moved inside the function definition as well?

Reply:
No it can't. The global variable won't work that way.